packages/ethereum-storage/src/ethereum-utils.ts

```diff
@@ -19,6 +19,8 @@ export default {
       [StorageTypes.EthereumNetwork.MAINNET as StorageTypes.EthereumNetwork]: 'mainnet',
       [StorageTypes.EthereumNetwork.KOVAN as StorageTypes.EthereumNetwork]: 'kovan',
       [StorageTypes.EthereumNetwork.RINKEBY as StorageTypes.EthereumNetwork]: 'rinkeby',
+      [StorageTypes.EthereumNetwork.XDAI as StorageTypes.EthereumNetwork]: 'xdai',
```

Comment:
If we add `xdai`, I would also add `sokol` as a testnet.

Reply:
true , actually i added in one of the ancient branches but the PR corresponding to that was closed . i will do the needful.